stages/org.osbuild.kernel-cmdline.bls-append

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+"""
+Add kernel command line parameters to a BLS [1] config either in
+the tree or in a mount.
+
+[1] https://freedesktop.org/wiki/Specifications/BootLoaderSpec/
+"""
+
+
+import glob
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": false,
+  "required": ["kernel_opts"],
+  "properties": {
+    "kernel_opts": {
+      "description": "Additional kernel command line options",
+      "type": "array",
+      "items": {
+        "description": "A single kernel command line option",
+        "type": "string"
+      }
+    }
+  }
+},
+"devices": {
+  "type": "object",
+  "additionalProperties": true
+},
+"mounts": {
+  "type": "array"
+}
+"""
+
+
+def main(paths, tree, options):
+    kopts = options.get("kernel_opts", [])
+
+    # If the user specified mounts that mounted
+    # in /boot then they want to operate on a mounted
+    # disk image rather than the tree. If not then use
+    # the tree.
+    mountroot = paths["mounts"]
+    boot = os.path.join(mountroot, "boot")
```

Comment:
Can this actually be hardcoded to "/boot" or should we communicate this in a different way? I know it's semi-default but my systems don't have a `/boot` in some cases (but only an `/efi`).

Reply:
I agree.  Let's add an option for `bls_path` (or simply `path`?) and have it default to `/boot`.